ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/notification/NotificationManagerImpl.java

```diff
@@ -100,8 +100,8 @@ public NotificationManagerImpl(NotificationManagerView view,
 
     @Inject
     @PostConstruct
```

Comment:
Off-top question: this is the first place on the client where annotation `@PostConstruct` has been used. Do we really need it?